src/libstore/build/local-derivation-goal.cc

```diff
@@ -205,6 +207,17 @@ void LocalDerivationGoal::tryLocalBuild() {
         #endif
     }
 
+    #if __linux__
+    if (useChroot) {
+        if (!mountNamespacesSupported() || !pidNamespacesSupported()) {
+            if (!settings.sandboxFallback)
+                throw Error("this system does not support the kernel namespaces that are required for sandboxing");
```

Comment:
This removes the mention of `sandbox-fallback` that was here in the previous error message, which I guess would make it harder to discover

Reply:
Added a mention of `--no-sandbox` in the error message.